homeassistant/components/netatmo/sensor.py

```diff
@@ -322,6 +322,10 @@ def _create_sensor_entity(netatmo_device: NetatmoDevice) -> None:
 
     @callback
     def _create_room_sensor_entity(netatmo_device: NetatmoRoom) -> None:
+        if not netatmo_device.room.climate_type:
+            msg = f"No climate type found for this room: {netatmo_device.room.name}"
```

Comment:
Why log this at info level? The message doesn't seem user facing. Lower it to debug level.